pkg/perfcounter/counter_logexporter.go

```diff
@@ -52,5 +54,11 @@ func (c *CounterLogExporter) Export() []zap.Field {
 
 	fields = append(fields, zap.Any("disk hit rate", float64(diskHits)/float64(diskReads)))
 
+	// all fields in CounterSet
+	_ = c.counter.IterFields(func(path []string, counter stats.Counter) error {
+		fields = append(fields, zap.Any(strings.Join(path, "."), counter.Load()))
```

Comment:
Previously we were calling `counter.SwapW(0)` to reset the local counter. 
`counter.Load()` won't reset the local counter.

If we want to print the global counter value in the logs, we can use Load(). If we want to log the counter value for the time interval, we should use SwapW(0). 

Note that, SwapW(0) won't affect the Global Counter within the stats counter.